### PR TITLE
Feature/xw 90/fix bijlagen

### PR DESCRIPTION
**Pull request by bbrands02**

### CODE REVIEW

File: src/Service/SyncXxllncCasesService.php [hunk 1]
```diff
@@ -181,34 +181,48 @@ private function deleteNonExistingObjects(array $idsSynced, Source $source, stri
...
+     *
+     * @return array  $fileURLS The view urls for files.
+     */
+    private function retrieveFile(array $result, array $documentMeta, array $config): array
+    {
+        // There can be expected here that there always should be a Bijlage ObjectEntity because of the mapping and hydration + flush that gets executed before this function.
+        // ^ Note: this is necessary so we always have a ObjectEntity and Value to attach the File to so we don't create duplicated Files when syncing every 10 minutes.
+        $bijlageObject = $this->entityManager->getRepository('App:ObjectEntity')->findByAnyId($documentMeta['uuid']);
+        $mimeType      = $documentMeta['mimetype'];
+        $base64        = $this->fileService->getInhoudDocument($result['id'], $documentMeta['uuid'], $mimeType, $config['source']);
```
Comment (rjzondervan) [inline src/Service/SyncXxllncCasesService.php:199]: Could you divide this line in multiple lines?

File: src/Service/SyncXxllncCasesService.php [hunk 2]
```diff
@@ -181,34 +181,48 @@ private function deleteNonExistingObjects(array $idsSynced, Source $source, stri
...
+
+        // This finds the existing Value or creates a new one.
+        $value = $bijlageObject->getValueObject("URL_Bijlage");
+        $this->entityManager->persist($value);
+
+        $url = $this->fileService->createOrUpdateFile($value, $documentMeta['filename'], $base64, $mimeType, $config['endpoint']);
```
Comment (rjzondervan) [inline src/Service/SyncXxllncCasesService.php:205]: Same here

File: Installation/Schema/woo.Bijlage.schema.json [hunk 3]
```diff
@@ -2,7 +2,7 @@
   "title": "Bijlage",
   "$id": "https://commongateway.nl/pdd.bijlage.schema.json",
   "$schema": "https://docs.commongateway.nl/schemas/Entity.schema.json",
-  "version": "0.0.1",
+  "version": "0.0.2",
   "type": "object",
   "description": "",
   "required": ["Titel_Bijlage", "URL_Bijlage"],
```
Comment (smisidjan) [inline Installation/Schema/woo.Bijlage.schema.json:8]: The URL_Bijlage can be removed from the array

**Review by smisidjan — CHANGES_REQUESTED** [review]

**Review by rjzondervan — COMMENTED** [review]
Seems alright, however, I don't see how you prevent double attachments.

**Pull request merged**